### DISCUSSION TITLE
Change save folder, and permanent saving/importing...

### DISCUSSION THREAD

**Discussion by marcogaio** [nroduit/Weasis#387] [Q&A]
I've tried to install and use weasis, on some Win10 64bit test machine. impressive!

I've two questions:

1) default folder for save data is $HOME\.weasis, that in Windows is the user profile; if (as we) there's roaming profiles enabled, this can be problematic; i've tried to look how move the defaut dir somewhere, but it is not clear to me...  seems tome could be weasis.portable.dicom.directory property, but i'll ask for confirmation.

2) we have tried to import a couple of CDs, using DICOMDIR, and thicking 'Copy images into the local temporary directory', and import worked as expected, and the '.weasis' dir get populated. But if we close the program and restart it, it is completely empty.
There's no way to import 'permanently' images?

Thanks.

**Comment by nroduit** [comment]
1) No the `.weasis` folder is attached to the Java System property "user.home" which is the user home directory. If this folder is stored remotely then it may slow down the launch. It is also possible to change this property in different ways (see [this page](https://nroduit.github.io/en/basics/customize/preferences/)).

2) It is by design that Weasis removes all temporary or downloaded data when the application is closed. Otherwise, you need to export the data locally.

**Reply by marcogaio** [reply]
> 1. No the `.weasis` folder is attached to the Java System property "user.home" which is the user home directory. If this folder is stored remotely then it may slow down the launch. It is also possible to change this property in different ways (see [this page](https://nroduit.github.io/en/basics/customize/preferences/)).

OK, thanks.
 
>     2. It is by design that Weasis removes all temporary or downloaded data when the application is closed. Otherwise, you need to export the data locally.

Oh... i supposed that WEASIS can manage a 'local archive' of patient/images; seems to me that instead need a remote database/server of patient data to connect to.

There's no way to manage a local DB? Thanks.
